src/parse/Node.zig

```diff
@@ -329,6 +329,11 @@ pub const Tag = enum {
     /// Example: EXAMPLE
     /// * lhs - LHS DESCRIPTION
     /// * rhs - RHS DESCRIPTION
+    multiline_string,
+    /// DESCRIPTION
+    /// Example: EXAMPLE
+    /// * lhs - LHS DESCRIPTION
+    /// * rhs - RHS DESCRIPTION
```

Comment:
I think we should either declare bankruptcy and get rid of these or actually start filling them in. Repeating this boilerplate is just silly.